### PR TITLE
[G2M] communications - guides on meetings

### PR DESCRIPTION
**Pull request by woozyking**
dedicate a complete doc for guides on meetings.

transforms `communications.md` to `communications/README.md`

### CODE REVIEW

File: communications/meetings.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+
+## During A Meeting
+
+Objectives:
+- Maximize the gain of synchronous/in-person/voice communication to resolve items.
+- Record as many details as possible for future context.
```
Comment (YizhiCatherineZhang) [inline communications/meetings.md]: Need to think a better way of taking meeting notes. I don't think count on 1 or 2 people every time is a good idea. everyone should contribute to taking notes, at least on the topic related to him/her.

File: communications/meetings.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Before A Meeting
+
+Objectives:
+- Give all participants as much context as possible before the meeting.
+- _Asynchronously_ resolve as many items as possible before the meeting.
+- Determine, and possibly _reduce_, the scope, and length of the meeting.
+- Determine, and possibly _eliminate_, reasons to participate in the meeting.
+- Prepare the meeting set up to be as convenient as possible for all participants.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
```
Comment (YizhiCatherineZhang) [inline communications/meetings.md]: Does this one mean everyone should participate in taking meeting notes?  Don't fully understand this sentence.

File: communications/meetings.md (hunk 1 above)
Comment (woozyking) [reply communications/meetings.md]: that's exactly what the guide says.

Comment (woozyking) [reply communications/meetings.md]: if you read the guide from _before, during, after_ in context, it should be clear that meeting notes is effectively decomposed of:
1. Mostly pre-meeting discussions (by everyone relevant).
2. Some in-meeting takeaways (by everyone relevant).
3. Some after-meeting afterthoughts (by everyone relevant).

File: communications/meetings.md (hunk 2 above)
Comment (woozyking) [reply communications/meetings.md]: I left things open to avoid unnecessary contest on being too rigid on the process, and/or may-or-may-not applicable legal implications. The execution very much depends on the willpower of the actual participants.

Essentially, this should be a gateway toward less-to-no meetings. The pre-meeting gets the most attention conveys this point -- most meetings can be avoided. But if it doesn't achieve this purpose, at least it betters the conventional meeting set up drastically.

File: communications/meetings.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Guide:
+- Preferably 1 work-day before the meeting, initiate one discussion per agenda item.
```
Comment (maluhoss) [inline communications/meetings.md]: Recommend having two subtitles: Setting Agenda & Context for the first 2 points to make it easier. 

Also just tested out the new tool Hugo and it's pretty cool. One idea to encompass what you've written here is:
```
* Setting Agenda: Preferably 1 work-day before the meeting, initiate one discussion for agenda (instead of per agenda item)
    * Create agenda on Hugo for meeting & send to relevant slack channel or person
    * For a net-new meeting, the organizer(s) should define the initial scope of items.
    * For a recurring meeting with past sessions, use the agenda items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
    * Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
```

By sending agenda to participants in Slack from Hugo, they can start a thread in Slack and any comments on that thread will become a comment on the agenda in Hugo

File: communications/meetings.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    - The discussion can be a Slack thread, GitHub issue, Asana task, Google Docs, or anything with an interactive "comments" component.
+    - For a recurring meeting with past sessions, use the ones from the previous meeting session as a starting point. Remove ones no longer needed.
+    - For a net-new meeting, the organizer(s) should define the initial scope of items.
+    - Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
+- In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant.
```
Comment (maluhoss) [inline communications/meetings.md]: Continuation of previous comment, suggestion for this part is have subtitle plus additional part of participants commenting on agenda thread:

```
* Context: In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant as a comment on the agenda thread.
    * For a net-new meeting, organizer should include context to agenda items
    * For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each has done, has been doing, and plans to do. Give everyone else a sense of where you are and the direction you're heading to. This helps others to determine if there are potential directional conflicts/blocks.
    * Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
    * Preemptively discuss and seek to resolve as many issues and directional conflicts/blocks as possible.

``` 
May recommend providing an example here of how this workflow could look like. I can create a gif to illustrate this :)

File: communications/meetings.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+
+## During A Meeting
+
+Objectives:
+- Maximize the gain of synchronous/in-person/voice communication to resolve items.
```
Comment (maluhoss) [inline communications/meetings.md]: Need to define what is the actual gain for in-person communication. Suggestion for this part:
```
* Align & inform all participants of relevant information at the same time (synchronously)
* Maintain time for each agenda item & scope of meeting as intended
* Record as many details as possible for future context/meetings
* Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
```

File: communications/meetings.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+- Record as many details as possible for future context.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+- Control time usage.
+
+Guide:
+- Demonstrate issues, instead of just abstractly talking about them.
```
Comment (maluhoss) [inline communications/meetings.md]: Suggest including point about going over agenda items + time associated with them + order change + removal of point about aggressively timeboxing and leaving jokes aside (opinionated for the nature of this section):

```
* Briefly go over the agenda items along with time given for each item
* If certain items cannot be resolved within the timebox, record all reasons and context in meeting note in Hugo and move on.
* Demonstrate issues instead of just abstractly talking about them so participants may be able to provide targeted tips/information to address issue
* Proactively record notes in Hugo under event
```

File: communications/meetings.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+- Aggressively control the timebox of each item. If the item is resolved sooner than the tentative timebox, skip ahead promptly, leave the jokes and fun talks for some other time.
+- If certain items cannot be resolved within the timebox, record all reasons and context into the per-item discussion formed in [section: Before A Meeting](#before-a-meeting) and move on.
+- Proactively record notes in the per-item discussion formed in [section: Before A Meeting](#before-a-meeting).
+
+## After A Meeting
+
+Objectives:
+- Have clear takeaways.
```
Comment (maluhoss) [inline communications/meetings.md]: can combine comprehensive and clear takeways into one point. Also an idea for implementing habit for having documents in technical knowledge drive:

```
* organize meeting notes to have clear takeaways and be as comprehensive as possible, for both non-participants and future self.
* Determine suitable follow-ups for leftovers.
* Transfer relevant information into Technical Knowledge Drive
```

File: communications/meetings.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+- Organize meeting notes to be as comprehensive as possible, for both non-participants and future self.
+- Determine suitable follow-ups for leftovers.
+
+Guide:
+- Amend per-item discussion with afterthoughts.
+- Summarize and derive actionable from the per-item discussion, place them within the per-item discussion
+    - It may be a good idea to use a more distinguishable section for the summaries and actionable, highly dependent on what tool is used for the discussions (WIP: need suggestions).
+- Attempt to resolve leftovers in per-item discussion asynchronously.
+- If the scope of the leftover is too complex, set a time for a follow-up meeting.
+- Preserve per-item discussion as meeting notes for future reference.
```
Comment (maluhoss) [inline communications/meetings.md]: can remove the last point since Hugo will do that for us

File: communications/meetings.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+## After A Meeting
+
+Objectives:
+- Have clear takeaways.
+- Organize meeting notes to be as comprehensive as possible, for both non-participants and future self.
+- Determine suitable follow-ups for leftovers.
+
+Guide:
+- Amend per-item discussion with afterthoughts.
```
Comment (maluhoss) [inline communications/meetings.md]: Can combine amend & summarize into one point. Suggestion:

```
* Amend, summarize and derive actionable item for each agenda item
    * It may be a good idea to use a more distinguishable section for the summaries and actionable, highly dependent on what tool is used for the discussions (WIP: need suggestions).

```

File: communications/meetings.md (hunk 4 above)
Comment (woozyking) [reply communications/meetings.md]: please do create a gif

File: communications/meetings.md (hunk 6 above)
Comment (woozyking) [reply communications/meetings.md]: agree mostly. but going over agenda items part should be done before meeting in this setup, right?

File: communications/meetings.md (hunk 4 above)
Comment (maluhoss) [reply communications/meetings.md]: Gifs found in here: https://drive.google.com/drive/folders/124IuFXJXmx4M6jqCZWnRjnOMkZ4KJR8h?usp=sharing

File: communications/meetings.md (hunk 6 above)
Comment (maluhoss) [reply communications/meetings.md]: Yup individually but found it helpful to give an overview right at the beginning of the meeting and prime people about the length of time that will be spent on diff items. My observations during tech connect.

File: communications/meetings.md (hunk 7 above)
Comment (woozyking) [reply communications/meetings.md]: gonna leave the Technical Knowledge Drive slightly open to just mention a knowledge base. we can leave it to the actual implementation.

eventually, we want to form the knowledge base in a few places. one potentially served from github through a headless cms solution and presented through a static site framework such as gatsby.js

File: communications/README.md [hunk 10]
```diff
@@ -23,7 +23,7 @@ Minimize Distractions: Nowadays we have way too many channels of receiving infor
 
 Opt-out: Leave channels that you are no longer interested in or obligated to.
 
-Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted. 
+Asynchronous Communication by Default: Whenever you want to communicate with others, default to an asynchronous way. By doing so, you respect your coworker's time and their productivity and the fact that they may be in the "zone" and should not be interrupted.
```
Comment (maluhoss) [inline communications/README.md]: I just remembered that we could address this point a little differently. Maybe mention something about how some people prefer in-person chats. Suggestion:
'Asynchronous Communication by Default: Each person has their own communication preferences. However, unless the person has stated that they prefer in-person chats over asynchronous communication, default to an asynchronous way. By doing so....'

Comment (maluhoss) [inline communications/README.md]: I just remembered that we could address this point a little differently. Maybe mention something about how some people prefer in-person chats. Suggestion:
'Asynchronous Communication by Default: Each person has their own communication preferences. However, unless the person has stated that they prefer in-person chats over asynchronous communication, default to an asynchronous way. By doing so....'

File: communications/meetings.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+- Create the meeting agenda on Hugo for the meeting, provide sufficient time ahead of the meeting to accommodate all participants.
+    - For a net-new meeting, the organizer(s) should define the initial scope of agenda items.
+    - For a recurring meeting with past sessions, use the agenda/notes items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
+- Initiate one or more meeting note(s) on Hugo for meeting agenda discussions.
```
Comment (maluhoss) [inline communications/meetings.md]: Would recommend saying 'initiating one note for meeting agenda' to avoid confusion of meeting note.

File: communications/meetings.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+    - Send these notes to the relevant Slack channel.
+    - Edit meeting agenda to reflect the outcome of these discussions.
+- In each meeting note, participants write down briefing, information, knowledge, and ideas that are relevant.
```
Comment (maluhoss) [inline communications/meetings.md]: Need to mention about participants commenting on agenda note in Slack about briefing, information, etc.
Suggestion:
`On the slack message for the agenda note, participants comment with briefing, information, knowledge and ideas that are relevant`

Then would recommend having the tip about Hugo doing a good job at syncing notes right after the point 'Sending these notes to the relevant Slack channel' right after this point.

File: communications/meetings.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+    - For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each _has done_, _has been doing_, and _plans to do_. Give everyone else a sense of _where you are_ and _the direction you're heading to_. This helps others to determine if there are potential directional conflicts/blocks.
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
```
Comment (maluhoss) [inline communications/meetings.md]: Would recommend having a line about after agenda is set, initiate a new note on Hugo for meeting notes to be used during the meeting.

Suggestion:
'Once agenda is finalized, initiate a note on Hugo for meeting notes titled Meeting Notes and share with the relevant Slack channel'

File: communications/meetings.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+Objectives:
+- Align & inform all participants of relevant information at the same time (synchronously)
+- Maintain time for each agenda item & scope of meeting as intended
+- Record as many details as possible for future context/meetings
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Briefly go over the agenda items, and timebox given for each item.
+- Demonstrate issues, instead of abstractly talking about them.
+- If certain items cannot be resolved within the timebox, record all reasons and context into the notes formed in [section: Before A Meeting](#before-a-meeting) and move on.
```
Comment (maluhoss) [inline communications/meetings.md]: should be 'record all reasons and context into the note titled Meeting Notes formed in ...'

File: communications/meetings.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+- All participants to proactively reply to meeting notes formed in [section: Before A Meeting](#before-a-meeting) to record as much context as possible. Start new notes as needed, such as when there are topics distinct from the ones formed before.
```
Comment (maluhoss) [inline communications/meetings.md]: Should be 'to the note titled Meeting Notes formed in ....' to make it clear which one to modify

File: communications/README.md (hunk 10 above)
Comment (woozyking) [reply communications/README.md]: sry this was actually not in the scope of this PR, but just a on-save removal of an excessive trailing whitespace.

but i agree with your point, let's address it in a subsequent PR.

File: communications/meetings.md (hunk 11 above)
Comment (woozyking) [reply communications/meetings.md]: but that's the Note feature regardless, as Agenda on Hugo cannot be sync'ed to Slack as thread there.

the one or more is intended to accommodate for meetings with complex scopes, and breaking down notes per-item or big topic would be benefitial.

this confusion may need a screenshot/gif/video to iron out, as one sentence really doesn't serve the purpose

File: communications/meetings.md (hunk 12 above)
Comment (woozyking) [reply communications/meetings.md]: the Note feature from Hugo can be sync'ed to Slack (both ways), so I wouldn't limit where participants want to carry out the discussion (ultimately does not matter as much). I guess I can incorporate this point as a subpoint under this one.

File: communications/meetings.md (hunk 13 above)
Comment (woozyking) [reply communications/meetings.md]: my intention is to actually keep using the per-item notes to be used during meeting. i may need to further clarify/reword to emphasize that.

File: communications/meetings.md (hunk 14 above)
Comment (woozyking) [reply communications/meetings.md]: no, same as the other point

> my intention is to actually keep using the per-item notes to be used during meeting. i may need to further clarify/reword to emphasize that.

File: communications/meetings.md (hunk 15 above)
Comment (woozyking) [reply communications/meetings.md]: no, same as above

> my intention is to actually keep using the per-item notes to be used during meeting. i may need to further clarify/reword to emphasize that.

File: communications/meetings.md (hunk 12 above)
Comment (woozyking) [reply communications/meetings.md]: nvm, there's a point above already covered sending to Slack channel.

perhaps we need to organize this, as it's a bit too wordy.

File: communications/meetings.md (hunk 11 above)
Comment (maluhoss) [reply communications/meetings.md]: ahhh then I think the title of each note will be important. If you set an agenda like:
Agenda
- Facebook Connector
- LocusML

Then initiate a note for each agenda item titled: 
Agenda Item Discussion: Facebook Connector
- Updates:
- Issues:

Something like that to be very clear

File: communications/meetings.md (hunk 12 above)
Comment (maluhoss) [reply communications/meetings.md]: Oh right, yes I agree. then better to say:
On Hugo or on Slack, participants comment on the agenda note with briefing, information, knowledge and ideas that are relevant

File: communications/meetings.md (hunk 13 above)
Comment (maluhoss) [reply communications/meetings.md]: I think if you mix in meeting notes with the agenda discussion, it can get very confusing. May be better to create one note for meeting documentation. 

One possible workflow is that during the meeting, one person will update the meeting note. However, the rest of the participants can comment on 2 possible places:
a) the note dedicated to the agenda item
b) the meeting note
Better to choose one and not both. Then whoever is updating the meeting note, can transfer the comments from the agenda item into the meeting note. Does this sound feasible?

File: communications/meetings.md (hunk 11 above)
Comment (woozyking) [reply communications/meetings.md]: absolutely. the title portion can be very useful.

File: communications/meetings.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+- Determine the scope and minimum length of the meeting.
+- Give all participants as much context as possible to make an informed decision on RSVP.
+- Gather topics and issues for agenda items that are relevant to the scope of the meeting,
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
```
Comment (YizhiCatherineZhang) [inline communications/meetings.md]: Shall we add that how to use zoom google doc here for the reference?
https://docs.google.com/document/d/1WNqeFKXYv_ji_PUXBKq8nlD09ErqmB6VoWasPW-BUDI/edit?usp=sharing

**Comment by YizhiCatherineZhang** [comment]
All good. I like this version. 👍

File: communications/meetings.md (hunk 16 above)
Comment (woozyking) [reply communications/meetings.md]: yea, i can link it, but common is public facing.

so we either want to make this google docs public, or we carry the content into common

Comment (maluhoss) [reply communications/meetings.md]: I can make it public. just need @hyx131 's approval since it has her face in it lol

Comment (maluhoss) [reply communications/meetings.md]: alright she approved
